### ISSUE TITLE
Library "ImportError" - Missing packages in setup.py

### ISSUE THREAD

**Issue by Joffreybvn** [enochkan/torch-metrics#21]
Following the installation instructions on the README, I get this error when trying:

`from torch_metrics import Accuracy`
`from torch_metrics import classification`

The only thing that work is:
`import torch_metrics`

**See the error**:
![Screenshot from 2021-02-11 10-18-11](https://user-images.githubusercontent.com/17571457/107619629-c0f06f00-6c53-11eb-978a-c69414b145a1.png)

It seems that the library's submodules are missing from `setup.py`.

**Comment by Joffreybvn** [comment]
Corrected on PR https://github.com/enochkan/torch-metrics/pull/22

People can temporarily install the lib with:
`pip install -i https://test.pypi.org/simple/ torch-metrics-corrected==1.1.7`

and import classes like this:
`from torch_metrics.classification import Accuracy`

**Comment by enochkan** [comment]
@Joffreybvn sorry work's been overwhelmingly busy. Thank you for your contribution. Closing this issue now.
